compiler/src/dotty/tools/dotc/core/TypeErrors.scala

```diff
@@ -17,7 +17,8 @@ import config.Printers.cyclicErrors
 
 class TypeError(msg: String) extends Exception(msg) {
   def this() = this("")
-  def toMessage(implicit ctx: Context): Message = getMessage
+  def toMessage(implicit ctx: Context): Message = super.getMessage
+  override def getMessage: String = throw new Exception("Use toMessage instead for TypeError")
```

Comment:
This may be annoying but at least no error messages will be empty due to this bug in the future

Reply:
Indeed, it helped find the problem in `Formatting.scala`. Thanks for suggesting it 👍